pandas/io/excel/_base.py

```diff
@@ -982,7 +1000,15 @@ class ExcelWriter(Generic[_WorkbookT]):
         (e.g. 'YYYY-MM-DD HH:MM:SS').
     mode : {{'w', 'a'}}, default 'w'
         File mode to use (write or append). Append does not work with fsspec URLs.
-    {storage_options}
+    storage_options : dict, optional
+    Extra options that make sense for a particular storage connection, e.g.
```

Comment:
For this and the two instance below, can you indent the body description as is done for other parameters.